extension/source/Controllers/Network/createEthMiddleware.ts

```diff
@@ -6,7 +6,17 @@ import {
   JRPCResponse,
 } from '@toruslabs/openlogin-jrpc';
 import { BigNumberish, BytesLike } from 'ethers';
-import { PROVIDER_JRPC_METHODS } from '../../common/constants';
+import web3_clientVersion from './web3_clientVersion';
+
+type ProviderHandler = (req: JRPCRequest<unknown>) => Promise<unknown>;
```

Comment:
This is likely beyond the scope of this work, but would be nice if we could genericize this vs. using `unknown`.

```suggestion
type ProviderHandler<ReqParams, ResResult> = (req: JRPCRequest<ReqParams>) => Promise<ResResult>;
```

Reply:
done